index.html

```diff
@@ -349,6 +349,28 @@ <h2>About Myself</h2>
 
 
 
+      </section>
+      <section class="contact-form">
+        <h2>Contact Me</h2>
+        <p>If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding.
+          I’d love to help with it!</p>
+        <form action="https://formspree.io/f/xoqzgvow" method="post"
+          enctype="multipart/form-data">
+          <input type="text" placeholder="Full names"
+            maxlength="30" value="Lorem ipsum" minlength="10"
+            required>
+          <input type="email" name="email" placeholder="email"
+            value="loremispum@gmail.com"
+            required>
+          <textarea rows="7" name="message"
```

Comment:
- [ ] As suggested by the previous reviewer, please kindly add a max length attribute with a value of 500 to the textarea. You may have mistakely added it to your button input instead. please fix this.